init_test.go

```diff
@@ -31,10 +31,10 @@ func TestAcceptance(t *testing.T) {
 	Expect(err).ToNot(HaveOccurred())
 
 	stack.BuildArchive = filepath.Join(root, "build", "build.oci")
-	stack.BuildImageID = fmt.Sprintf("stack-build-%s", uuid.NewString())
+	stack.BuildImageID = fmt.Sprintf("localhost:8000/stack-build-%s", uuid.NewString())
```

Comment:
Why do we need this now?

Reply:
See your next comment, they are related.